handbook/direction/maturity/index.md

```diff
@@ -0,0 +1,63 @@
+# Feature Maturity Tracker
+
+Sourcegraph keeps track of how mature each of our feature areas are, to help you make the best decisions about how to use our product. Clicking on the feature name will take you to its documentation.
+
+If you do not see content below, check you have JavaScript enabled then contact Product Management.
+
+## Maturity levels
+
+- ○: not yet available, but on our roadmap
+- ◔: available in the product, but may not be ready for production use yet
```

Comment:
Curious which products you think would go here?
If I am the PM of a team, I would be very hesitant to put my product here because "not ready for production use" will likely deter customers from trying it out and implies a _risk_ that may not actually be realistically there, even if I _don't_ see the _next_ level already being true for my product ("usable for primary jobs to be done" -> maybe we don't actually support all primary jobs yet, but we _are_ confident about the reliability of what we support, there being little _risk_ and it therefor being "production-ready").

I guess my thought overall here (considering my comments below too) is it feels really hard and potentially dangerous to map all of our products into this one linear maturity scale.

Reply:
These would match our existing definitions of beta or experimental, linking to that from here.

```suggestion
- ◔: available in the product, but may not be ready for production use yet (i.e., [experimental or beta features](../../product/beta_and_experimental_feature_labels))
```